doc/contributing.md

```diff
@@ -0,0 +1,127 @@
+
+The team triages open issues several times a week. During triage, the team uses labels to categorize, manage, and drive the project lifecycle. 
+
+The point of doing all this work in public is to ensure that we are holding ourselves to a high degree of transparency, and so that the community sees that we apply the same level of review and scrutiny to our issues and PRs as we do to community-submitted issues and PRs. We also want to make sure that we expose our team culture and "tribal knowledge" that is inherent in any closely-knit team, which often contains considerable value to those new to the project who are trying to figure out "why the heck does this thing look/work like this???"
+
+## Repo Bot
+
+We employ [a bot engine](https://github.com/microsoft/terminal/blob/master/doc/bot.md) to help us automate common processes within our workflow.
+
+We drive the bot by tagging issues with specific labels which cause the bot engine to close issues, merge branches, etc. This bot engine helps us keep the repo clean by automating the process of notifying appropriate parties if/when information/follow-up is needed, and closing stale issues/PRs after reminders have remained unanswered for several days.
+
+Therefore, if you do file issues, or create PRs, please keep an eye on your GitHub notifications. If you do not respond to requests for information, your issues/PRs may be closed automatically.
+
+## Reporting an bug/issue
+
+### DO NOT post "+1" comments
+
+⚠ DO NOT post "+1", "me too", or similar comments: If you don't have any additional info/context to add but would like to indicate that you're affected by the issue, upvote the original issue by clicking its [+😊] button and hitting 👍 (+1) icon. This way we can actually measure how impactful an issue is.
+
+Search existing issues
+Be sure to search existing open and closed issues before filing a new issue: Nobody enjoys reading 20 reports of the same issue - we'd MUCH prefer it if you add comments to an existing issue describing the same/similar issue.
+
+This project is moving fast and its likely someone else has found the problem you're seeing, and someone may be working on or have already contributed a fix!
+
+If not, great - please file a new issue:
+
+### File a new Issue
+
+When you hit "New Issue", select the type of issue closest to what you want to report/ask/request:
+!(/docs/images/new-issue-template.png)
+
+**Complete the requested information and provide as much information as possible**. The more information you provide, the more likely your issue/ask will be understood and fixed. Wherever possible, provide steps to reproduce the issue: **Repro steps are among the most valuable part of a bug report!**
+
+Other helpful information includes:
+
+* What device you're running (inc. CPU type, memory, disk, etc.)
+* What build of Windows your device is running
+  
+  👉 Tip: run `ver` in PowerShell/Cmd, i.e.:

```

Comment:
`ver` still doesn't work in PowerShell. Is there a PowerShell cmdlet that can get the OS version?